pyvista/examples/downloads.py

```diff
@@ -7240,3 +7240,83 @@ def download_victorian_goblet_face_illusion(load=True):  # pragma: no cover
 _dataset_victorian_goblet_face_illusion = _SingleFileDownloadableDatasetLoader(
     'Victorian_Goblet_face_illusion/Vase.stl'
 )
+
+
+def download_reservoir(load=True):  # pragma: no cover
+    """Download the UNISIM-II-D reservoir model.
```

Comment:
![NITS](https://img.shields.io/badge/review-nits-inactive.svg?style=for-the-badge) What is the difference between UNISIM-II-D and  UNISIM-II?

Reply:
I am not sure. We could ask the original author of the data for review ? The vtu file is sourced from here https://github.com/pyvista/pyvista-support/issues/319

Reply:
It is just an interest, and there is no need for that. If an outside party points this out to us, let's respond to it :)